packages/hooks/package.json

```diff
@@ -5,8 +5,10 @@
     "main": "src/index.ts",
     "dependencies": {
         "@deriv/api": "^1.0.0",
-        "@deriv/stores": "^1.0.0",
         "@deriv/shared": "^1.0.0",
+        "@deriv/stores": "^1.0.0",
+        "history": "^5.0.0",
+        "react-router-dom": "^5.2.0",
```

Comment:
@thisyahlen-deriv Not sure if it's a good idea to make `@deriv/hooks` dependent on the router library 🤔

Reply:
Its for the test case